scripts/get_token.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# This script will connect to the vCD using username and password,
+# and show the header that cointains an authorization token.
+#
+user=$1
+password=$2
+org=$3
+IP=$4
+
+if [ -z "$IP" ]
+then
+    echo "Syntax $0 user password organization IP_address"
```

Comment:
Maybe there's an option to **not** duplicate this script which is already in go-vcloud-director? Maybe some way to pull it in into `vendor/`?

Reply:
In vendor it would not go unless moved explicitly, and we might create conflicts during `go mod tidy` operations.
One way would be to link to its address on GitHub.
Or we can remove it altogether and only use `make token` in this repository.